File: Entity/Operation/Gestion.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Entity.Operation
{
    public class Gestion
    {
        public int IdFile { get; set; }
        public string Uuid { get; set; }
        public string RfcEmisor { get; set; }
        public string NombreEmisor { get; set; }
        public string RfcReceptor { get; set; }
        public string NombreReceptor { get; set; }
        public DateTime FechaCertificacionSat { get; set; }
        public float Monto { get; set; }
        public string EfectoComprobante { get; set; }
        public string Status { get; set; }
        public DateTime FechaCancelacion { get; set; }
        public string FileName { get; set; }
        public DateTime DateCreation { get; set; }
    }
}

File: Portal/ClientApp/src/app/models/Gestion.ts
import { Catalog } from "./Catalog";

export class Gestion {
  public idFile: number;
  public uuid: string;
  public rfcEmisor: string;
  public nombreEmisor: string;
  public rfcReceptor: string;
  public nombreReceptor: string;
  public fechaCertificacionSat: Date;
  public monto: number;
  public efectoComprobante: string;
  public status: string;
  public fechaCancelacion: Date;
  public fileName: string;
  public dateCreation: Date;
  public idTipoPago: number;
  public tipoPagoDescription: string;
  public idFormaPago: number;
  public formaPagoDescripcion: string;

  public tipoPago: Catalog;
  public formaPago: Catalog;
}
